src/main/scala/za/co/absa/hyperdrive/trigger/configuration/application/NotificationConfig.scala

```diff
@@ -28,5 +30,9 @@ class NotificationConfig (
   val enabled: Boolean,
   @DefaultValue(Array(""))
   @Name("sender.address")
-  val senderAddress: String
+  val senderAddress: String,
+  @DefaultValue(Array("0ms"))
+  val delay: Duration,
```

Comment:
Could you please add new props into PR description ? It is easier to collect all props changes during release

Reply:
Done